server/engine/src/main/java/org/eclipse/lsp/cobol/service/copybooks/CopybookServiceImpl.java

```diff
@@ -178,49 +172,18 @@ public void store(CopybookModel copybookModel, boolean doCleanUp) {
 
   private CopybookModel resolveSync(
       @NonNull CopybookName copybookName,
-      @NonNull String programUri,
-      @NonNull CopybookConfig copybookConfig) {
+      @NonNull String programUri) {
     ThreadInterruptionUtil.checkThreadInterrupted();
     LOG.debug(
-        "Trying to resolve copybook {} for {}, using config {}",
+        "Trying to resolve copybook {} for {}",
         copybookName,
-        programUri,
-        copybookConfig);
+        programUri);
     Optional<CopybookModel> copybookModel = tryResolveCopybookFromWorkspace(copybookName, programUri);
-    if (copybookModel.isPresent()) {
-      return copybookModel.get();
-    }
-    Optional<CopybookModel> predefineCopybook = tryResolvePredefinedCopybook(copybookName, copybookConfig);
```

Comment:
Why should cobol engine provide a predefined copybook. Let the dialects handle it. In any case if we need any special variables to be inserted, we could use processors to insert the variables in the AST.